src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs

```diff
@@ -44,6 +44,7 @@ internal partial class X11XamlRootHost : IXamlRootHost
 	private readonly ApplicationView _applicationView;
 	private readonly X11WindowWrapper _wrapper;
 	private readonly Window _window;
+	private readonly CompositeDisposable _disposables = new();
```

Comment:
This doesn't need to be a CompositeDisposable. just a SerialDisposable or a simple IDisposable would be fine.

Reply:
I don't have strong opinions on this, so it's fine to leave it as is.